vagrant/Vagrantfile

```diff
@@ -25,6 +26,8 @@ Vagrant.configure("2") do |config|
     yum -y install epel-release
     yum -y install centos-release-scl
     yum -y install yum-utils
+    yum -y install boost-devel
```

Comment:
These two should be removed. They are installed by "yum-builddep -y /vagrant/dist/vespa.spec" further down.